Add Viam integration page (#33925)
home-assistant/home-assistant.github.io: source/_integrations/viam.markdown

---
title: Viam
description: Support for Viam data and vision services using the Python SDK
ha_category:
  - Image Processing
ha_release: '2023.11'
ha_iot_class: Cloud Polling
ha_config_flow: true
ha_codeowners:
  - '@hipsterbrown'
ha_domain: viam
ha_integration_type: integration
---

> The Viam integration allows you to turn your smart home into a smart machine! Use images and data from your Home Assistant setup to train custom machine learning models that run on device as part of your automation workflow.

Reviewer: ```suggestion
The **Viam** {% term integrations %} allows you to turn your smart home into a smart machine! Use images and data from your Home Assistant setup to train custom machine learning models that run on device as part of your automation workflow.
```

## Prerequisites

In order to use this integration, you will need a [Viam account](https://www.viam.com/) and [a device with `viam-server` installed](https://docs.viam.com/installation/).

For authentication, you can use an [organization API key](https://docs.viam.com/manage/cli/#create-an-organization-api-key) or [robot address and location secret](https://docs.viam.com/manage/fleet/#control-with-the-sdks).

To use the classification and detection services, you will need a configured [vision service](https://docs.viam.com/services/vision/) for the integration.

{% include integrations/config_flow.md %}

> ## Services

Reviewer: ```suggestion
## Actions
```
In this release, we are renaming services to actions

> ## Services

Reviewer: ```suggestion
## Actions
```
